iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -547,26 +547,7 @@ TEST_F(PortManager_test, SubscriberRequiringHistorySupportDoesConnectToPublisher
     EXPECT_TRUE(publisher.hasSubscribers());
 }
 
-TEST_F(PortManager_test, SubscriberRequiringHistorySupportDoesNotConnectToPublisherWithInsufficientHistorySupport)
```

Comment:
obsolete case, now any history capacity > 0 matches and this is checked in e.g. SubscriberRequiringHistorySupportDoesConnectToPublisherWithLowerHistorySupport